src/Core/Checkout/Customer/Event/CustomerSetDefaultBillingAddressEvent.php

```diff
@@ -0,0 +1,96 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Checkout\Customer\Event;
+
+use Shopware\Core\Checkout\Customer\CustomerDefinition;
+use Shopware\Core\Checkout\Customer\CustomerEntity;
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\Event\BusinessEventInterface;
+use Shopware\Core\Framework\Event\EventData\EntityType;
+use Shopware\Core\Framework\Event\EventData\EventDataCollection;
+use Shopware\Core\Framework\Event\SalesChannelAware;
+use Shopware\Core\Framework\Event\ShopwareSalesChannelEvent;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+use Symfony\Contracts\EventDispatcher\Event;
+
+class CustomerSetDefaultBillingAddressEvent extends Event implements BusinessEventInterface, SalesChannelAware, ShopwareSalesChannelEvent
+{
+    public const EVENT_NAME = 'checkout.customer.default.billing.address.event';
+
+    /**
+     * @var CustomerEntity
+     */
+    private $customer;
+
+    /**
+     * @var SalesChannelContext
+     */
+    private $salesChannelContext;
+
+    /**
+     * @var string
+     */
+    private $contextToken;
+
+    private $addressId;
+
+    public function __construct(SalesChannelContext $salesChannelContext, CustomerEntity $customer, $addressId)
```

Comment:
please add type hint `string` to the address ID
also to the properties and methods